File: arkts_static_test/03_Types/ets_cases/3.17.2_Type_Tuple/cross_lang_verify/runtime/TYP_03_17_02_009.swift
/**
 * Swift cross-language verification for TYP_03_17_02_009_RUNTIME_INSTANCEOF_TUPLE
 */
import Foundation

let pair: (Int, String) = (1, "abc")

if !(pair is Any) {
    fatalError("pair should be Any")
}

print("verified")

File: arkts_static_test/03_Types/ets_cases/3.17.2_Type_Tuple/cross_lang_verify/runtime/TYP_03_17_02_009.java
/**
 * Java cross-language verification for TYP_03_17_02_009_RUNTIME_INSTANCEOF_TUPLE
 */
class TYP_03_17_02_009 {
    public static void main(String[] args) {
        Object[] pair = {1, "abc"};

        if (!(pair instanceof Object[])) {
            throw new AssertionError("pair should be Object[]");
        }

        System.out.println("verified");
    }
}
